tests/drivers/input/gpio_kpp/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
```

Comment:
this should go in `samples/boards/<vendor>` if it's not usable on other vendor's SoCs